app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :user
+  validates :title, presence: true
+  validates :body, presence: true
+  validates :user, presence: true
```

Comment:
- [ ] **[REQUIRED]** Presence is being checked two times. Usually `belongs_to` would be enough in this situation. When I try to save an invalid post using rails console, I get this:
```ruby
irb(main):016:0> p1 = Post.new
irb(main):017:0> p1.save
=> false
irb(main):018:0> p1.errors.full_messages
=> ["User must exist", "User can't be blank", "Title can't be blank", "Body can't be blank"]
```
Notice, that its checking twice for user. Please fix it.